docs/tutorials/multinode/terraform/init.sh

```diff
@@ -0,0 +1,29 @@
+# ---- Edit below -----#
+
+export TF_VAR_project=quantum-htcondor-15
+export TF_VAR_project_id=us-east4-c
+export TF_VAR_zone=us-east4-c
+export TF_VAR_region=us-east4
+export TF_VAR_multizone=true
+export TF_VAR_numzones=4  # for regional/multizone, set to the number of regions in the zone
+# ---- Do not edit below -----#
+
+export TF_VAR_project_id=${TF_VAR_project}
```

Comment:
@jrossthomson Is this overwriting the initial variable assignment on line 4? (asking because I need to modify this to use gcp images)